go/v1beta1/storage/filtering/filtering.go

```diff
@@ -114,6 +121,17 @@ func parseExpression(expression *expr.Expr) (*Query, error) {
 				},
 			},
 		}, nil
+	case overloads.StartsWith:
+		leftTerm, rightTerm, err := getSimpleExpressionTerms(leftArg, rightArg)
+		if err != nil {
+			return nil, err
+		}
+
+		return &Query{
+			Prefix: &Term{
```

Comment:
This is the shortened version of Elasticsearch's [prefix query](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-prefix-query.html#prefix-query-short-ex)